pkg/client/record/events_cache.go

```diff
@@ -19,6 +19,7 @@ package record
 import (
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/api"
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/util"
+	"github.com/golang/groupcache/lru"
 	"sync"
 )
 
```

Comment:
History probably didn't need to be exposed outside the package?  Consider making it lower-case?